Disable roundeding corners for token placeholder icons #4656

oa-s opened the pull request:
Closes #4656

hboon commented:
@oa-s can you help attach screenshots?

Are the borders embedded in the images or do we add them dynamically?

oa-s commented:
i moved out from dynamyc rounding of image, and replaced with rounding of image view. a place holder its other image view over web Image view, that is not aplied by rounding.

hboon reviewed (approved):
Building tests seems to fail

oa-s commented:
After PR:
<img width="511" alt="Screenshot 2022-05-27 at 08 16 56" src="https://user-images.githubusercontent.com/96283950/170634238-c2743337-0d30-4ece-a192-13ab358ae16f.png">
<img width="511" alt="Screenshot 2022-05-27 at 08 16 50" src="https://user-images.githubusercontent.com/96283950/170634251-588d4ab5-20f3-42af-a052-d0e63415b10f.png">
<img width="511" alt="Screenshot 2022-05-27 at 08 16 45" src="https://user-images.githubusercontent.com/96283950/170634254-5dc543fb-7a45-430d-93cf-bc4bda3c2017.png">
<img width="511" alt="Screenshot 2022-05-27 at 08 16 39" src="https://user-images.githubusercontent.com/96283950/170634260-b25e6408-67ed-4692-884e-db1f221bcebb.png">
